migrations/postgres/2020-01-05-232816_add_blocklist/up.sql

```diff
@@ -0,0 +1,6 @@
+-- Your SQL goes here
+create table email_blocklist(id SERIAL primary key,
```

Comment:
Little detail, so if you don't feel like doing it, don't change, but: keywords in SQL usually are in all caps.

Reply:
Might as well follow the conventions :)